tests/data/rawvoice_donate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Namespace Test",
+  "episodes": [],
+  "donate": "Support Podcast Name"
```

Comment:
It doesn't really make sense to just have the text here, according to the spec:

<q>donate, if present as a sub-element of <channel>, allows for syndicating a donate page for crowdfunding purposes. It has one required attribute href. A value may be specified but is not required.</q>

So this should probably be something like:

```
    ...
    "donate": {
        "text": "Support Podcast Name",
        "href": "http://example.com/feed/podcast/"
    }
    ...
```